src/Groups/iso_oscar_gap.jl

```diff
@@ -0,0 +1,103 @@
+import AbstractAlgebra: MatElem
+import Hecke: FqNmodFiniteField, FqNmodMatSpace, fq_nmod, fq_nmod_mat
+import GAP: FFE
+
+# isomorphism from the Oscar field to the GAP field
+struct GenRingIso <: Map{FqNmodFiniteField,GapObj,SetMap,GenRingIso}
+   domain::FqNmodFiniteField
+   codomain::GapObj
+   f
+   f_inv
+end
+
+# turns the Oscar matrix into the GAP matrix and viceversa
+struct GenMatIso <: Map{FqNmodMatSpace,GapObj,SetMap,GenMatIso}
+   domain::FqNmodMatSpace
+   codomain::GapObj
+   fr::GenRingIso
+   f
+   f_inv
+end
+
+
+################################################################################
+#
+#  Ring isomorphism
+#
+################################################################################
+
+function elem_f(x::fq_nmod; B=0,d=0)
+   COEF = [Int64(coeff(x,i)) for i in 0:d-1]
+   x_gap = sum([COEF[i]*B[i] for i in 1:d])
+   return x_gap
+end
+
+function elem_g(x::FFE; B=0, z=0, d=0)
+   L = GAP.gap_to_julia(GAP.Globals.List(GAP.Globals.Coefficients(B,x),GAP.Globals.IntFFE ))
```

Comment:
What's the output type? a vector of ints? Then I'd recommend something like this instead of using `gap_to_julia`:
```suggestion
   L = Vector{Int}(GAP.Globals.List(GAP.Globals.Coefficients(B,x),GAP.Globals.IntFFE ))
```